src/sentry/tsdb/snuba.py

```diff
@@ -25,11 +25,11 @@ class SnubaTSDB(BaseTSDB):
     model_columns = {
         TSDBModel.project: ('project_id', None),
         TSDBModel.group: ('issue', None),
-        TSDBModel.release: ('release', None),
+        TSDBModel.release: ('sentry:release', None),
         TSDBModel.users_affected_by_group: ('issue', 'user_id'),
         TSDBModel.users_affected_by_project: ('project_id', 'user_id'),
```

Comment:
Unrelated, but I think these refer to the tag value in the Redis TSDB backend. We'll figure this out in result comparison, I guess?